tests/functions.php

```diff
@@ -0,0 +1,35 @@
+<?php
+require '../functions.php';
+use PHPUnit\Framework\TestCase;
+
+class functions extends TestCase {
+    public function testIsStringDisplayAlbums() {
```

Comment:
need to write another success test that does an assert Equals

Reply:
done